src/components/InsightWidget.tsx

```
@@ -430,27 +468,279 @@ export const InsightWidget: React.FC<InsightWidgetProps> = ({ className, onRemov
-      setMarketSentiment(analyzeMarketSentiment(allDigests[newIndex].content));
+  // Set up anime.js animations
+  useEffect(() => {
+    // Create a new scope for animations
+    if (containerRef.current) {
+      animeScope.current = createScope({ root: containerRef.current });
     }
-  };
+    
+    // Set up cleanup
+    return () => {
+      if (animeScope.current && animeScope.current.revert) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)